index.html

```diff
@@ -1 +1,941 @@
+                                </article>
+                              </div>
+
+                              <div class="clearfix more-stories-part">
+                                <article>
+                                  <div class="image"><img src="https://d.newsweek.com/en/full/1513442/big-penis.jpg?w=397&h=265&f=b15a0cf4618087890653eb3505134d79" class="img-fluid" alt=""></div>
+                                  <h6>HFDA Warns 'Big Penis' Male Sexual Supplement May Be Too Powerful</h6>
+                                </article>
+                              </div>
+
+                              <div class="clearfix more-stories-part">
+                                <article>
+                                  <div class="image"><img src="https://d.newsweek.com/en/full/1513431/church-swastika-graffiti.jpg?w=397&h=265&f=594bf7c941cd98cab81fcec6a51c7b56" class="img-fluid" alt=""></div>
+                                  <h6>Swastika and Racial Slurs Spray-Painted on Tennessee Church</h6>
+                                </article>
+                              </div>
+
+                              <div class="clearfix more-stories-part">
+                                <article>
+                                  <div class="image"><img src="https://d.newsweek.com/en/full/1513413/russia-lgbtq-protest.jpg?w=397&h=265&f=326ffdf87c178fa44210aa357c7c80f6" class="img-fluid" alt=""></div>
+                                  <h6>Russia Blocks Website That Urges Readers to Hunt and Attack LGBTQ People</h6>
+                                </article>
+                              </div>
+
+                              <div class="clearfix more-stories-part">
+                                <article>
+                                  <div class="image"><img src="https://d.newsweek.com/en/full/1513420/alan-dershowitz-jeffrey-epstein-sex-underage-girls.jpg?w=397&h=265&f=36d64f24b8f29fec0ea841dd25a34ee3" class="img-fluid" alt=""></div>
+                                  <h6>Dershowitz Boasts of 'Perfect Sex Life' Amid Epstein Rape Allegations</h6>
+                                </article>
+                              </div>
+
+                          </section>
+
+                              <div class="block">
+                                  <div class="d-flex align-items-center" id="add">
+                                      <span class="p-2 align-self-center icon-n"></span>
+                                      <span class="p-2" id="block-text">Sign up for our Newsletter</span>
+                                  </div>
+
+                                  <span class="signup">SIGN UP</span>
+                                  <p class="update">Update your preferences »</p>
+                              </div>
+
+                          </div>
+                      </div>
+                  </div>
+              </div>
+
+              <section class="bottom">
```

Comment:
The homeless `</div>` from line 887 is causing lots of trouble. Might make it to the Newsweek headlines... =D